public/500.html

```diff
@@ -54,7 +54,7 @@ <h1>We're sorry, but something went wrong.</h1>
   </div>
   <p>
     If you are the application owner check the logs for more information.
-    <!-- AIRBRAKE ERROR -->
+    <!-- ERROR NOTIFIER ERROR -->
```

Comment:
maybe `ERROR NOTIFIER ERROR` -> `ERROR NOTIFIER`

Reply:
But it is technically an `ErrorNotifier` error placeholder?